nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -462,30 +462,39 @@ Here is an example entry:
 
 After you have edited and saved the 'authorizers.xml' file, restart NiFi. Users and roles from the 'authorized-users.xml' file are converted and added as identities and policies in the 'users.xml' and 'authorizations.xml' files.  Once the application starts, users who previously had a legacy Administrator role can access the UI and begin managing users, groups, and policies.
 
-Here is a summary of policies assigned to each legacy role if the NiFi instance has an existing flow.xml.gz:
+The following tables summarize the global and component policies assigned to each legacy role if the NiFi instance has an existing flow.xml.gz:
```

Comment:
It appears the standard is to note filenames via a single quote, which makes it italic. Suggest changing `flow.xml.gz` to `'flow.xml.gz'`.